service/src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -304,7 +304,7 @@ public ResponseEntity<ApiCreatedControlledGcpBigQueryDataset> createBigQueryData
             .datasetName(
                 ControlledBigQueryDatasetHandler.getHandler()
                     .generateCloudName(
-                        null,
+                        workspaceUuid,
```

Comment:
null. workspaceUuid  is not need

Reply:
Hi Yu. I am ok with both idea. In Zach's last comment, he suggests to mention workspaceUuid even it not been used.

Reply:
oh ok. i missed that. no need to change back.